internal/locate/region_request.go

```diff
@@ -1288,14 +1288,24 @@ func (s *replicaSelector) onServerIsBusy(
 	return true, nil
 }
 
-// For some reasons, the leader is unreachable by now, try followers instead.
-// the state is changed in accessFollower.next when leader is unavailable.
-func (s *replicaSelector) canFallback2Follower() bool {
+// canSkipServerIsBusyBackoff returns true if the request can be sent to next replica and can skip ServerIsBusy backoff.
+func (s *replicaSelector) canSkipServerIsBusyBackoff() bool {
```

Comment:
Maybe we can reuse this function handling region unavailable and changing it's name in the future. Not this PR's work.